data/postgres-seed.js

```diff
@@ -0,0 +1,40 @@
+const { db, pgp } = require('./postgres/index.js');
+const earnings = require('./earnings.js');
+const tickers = require('./tickers.js');
+
+const cs = new pgp.helpers.ColumnSet(['ticker', 'name', 'earnings'], { table: 'tickers' });
+
+const start = () => {
+  let tickerList = tickers.createTickers().map(ticker => {
+    return {
+      ticker: ticker,
+      name: earnings.generateName(),
+      earnings: earnings.generateEarningsList()
+    };
+  });
+  return tickerList;
+}
+
+async function seedDB() {
+  let total = 1000000
+  let count = 0;
+  let chunk = start().length;
+
+  while (count < total) {
+    count += chunk;
```

Comment:
from my understanding of influx threshold you could batch the tickers into every 5000... what made you choose 100?

Reply:
I chose 1000 for Postgres since that worked the best for me